Neos.ContentRepositoryRegistry/Classes/Configuration/NodeTypeEnrichmentService.php

```diff
@@ -120,26 +124,32 @@ protected function setPropertyLabels(string $nodeTypeName, array &$configuration
                 )
             ) {
                 $propertyConfiguration['ui']['inline']['editorOptions']['placeholder']
-                    = $this->getPropertyConfigurationTranslationId(
+                    = $this->getConfigurationTranslationId(
                     $nodeTypeLabelIdPrefix,
                     $propertyName,
-                    'ui.inline.editorOptions.placeholder'
+                    'ui.inline.editorOptions.placeholder',
+                    $configurationType
                 );
             }
 
             if (
                 isset($propertyConfiguration['ui']['help']['message'])
                 && $this->shouldFetchTranslation($propertyConfiguration['ui']['help'], 'message')
             ) {
-                $propertyConfiguration['ui']['help']['message'] = $this->getPropertyConfigurationTranslationId(
+                $propertyConfiguration['ui']['help']['message'] = $this->getConfigurationTranslationId(
                     $nodeTypeLabelIdPrefix,
                     $propertyName,
-                    'ui.help.message'
+                    'ui.help.message',
+                    $configurationType
                 );
             }
+            if (isset($propertyConfiguration['properties'])) {
+                $this->setLabels($nodeTypeName, $propertyConfiguration['properties'] , $superTypeConfigResolver, $propertyName.'.properties');
```

Comment:
```suggestion
                $this->setLabels($nodeTypeName, $propertyConfiguration['properties'], $superTypeConfigResolver, $propertyName . '.properties');
```

Reply:
but here `string $configurationType` doesnt make sense? It will be `$propertyName . '.properties'` but should be `'properties'` right?